eggsy.codes/dist/metadata.json

```diff
@@ -9,7 +9,8 @@
         "tr": "eggsy.codes, EGGSY'nin PreMiD ile ve bir çok başka projesiyle alakalı işlerini sergilediği internet sitesidir!"
     },
     "url": "eggsy.codes",
-    "version": "1.0.1",
+	"regExp": "^((?!custom-status).)*$",
+    "version": "1.0.2",
```

Comment:
Now it will work on every website unless it includes custom-status in its url, I suggest you make it only for eggsy.codes

Reply:
TIMERAA SAID IT LOOKS UP FOR REGEX FIRST AND THEN THE URL DUDE DON'T WORRY, IF IT PASSES THE REGEX IT'D STUCK WITH THE URL.